### ISSUE TITLE
[Suggestion] Harmless laser

### ISSUE THREAD

**Issue by ncrecc** [HugoBDesigner/Mari0-SE-Bugfixes#20]
A laser that's blue or whatever and lets Mario pass right through it. Can only be blocked by cubes, toggles all the same detectors as a normal laser.

**Comment by villager103** [comment]
There's an element from the Portal 1 mod "Blue Portals" that does basically the same thing as this; it's a light beam that acts the same as a laser, and it doesn't kill you. I think having the light beams would be better than adding a second laser type.

**Comment by HugoBDesigner** [comment]
Duplicate of #11
